src/proto/parameters.proto

```diff
@@ -126,9 +126,8 @@ message PassingConfig
 
 message GoalieTacticConfig
 {
-    // Control the final speed of the goalie to be able to intercept shots better
-    required double goalie_final_speed = 1
-        [default = 0.0, (bounds).min_double_value = 0.0, (bounds).max_double_value = 3.0];
+    reserved 1;
+
```

Comment:
How come we don't need this field with the trajectory planner?

Reply:
The bang bang trajectories don't support having a non-zero final velocity. Plus, Im pretty sure this field wasn't used